### ISSUE TITLE
stream data

### ISSUE THREAD

**Issue by ali-Eskandarian** [datamllab/tods#91]
%md
Does this package support streaming data in anomaly detection?

**Comment by lhenry15** [comment]
No, we only support batch setting. You may want to take a look at [Stream-AD](https://github.com/Stream-AD) for more streaming anomaly detection works.
